src/components/Form/SetAclPermsDocContainerForm.jsx

```diff
@@ -34,7 +34,7 @@ const SetAclPermsDocContainerForm = () => {
   const { session } = useSession();
   const { state, dispatch } = useStatusNotification();
   const { selectedUser } = useContext(SelectedUserContext);
-  const [username, setUsername] = useState(selectedUser.username);
+  const [username, setUsername] = useState('');
```

Comment:
Using empty string allows the application to reset username when clearing client.

https://github.com/codeforpdx/PASS/assets/14917816/bea20026-4706-4d9f-9722-1196b5d5fd92